Packs/Traceable/Integrations/Traceable/Traceable.py

```diff
@@ -311,8 +312,35 @@ def set_threat_category_list(self, threatCategoryList):
     def set_ip_reputation_level_list(self, ipReputationLevelList):
         self.ipReputationLevelList = ipReputationLevelList
 
+    def set_ip_categories_list(self, ipCategoriesList):
+        _ipCategoriesList = []
+        for ipCategory in ipCategoriesList:
+            if ipCategory == "Unknown":
+                _ipCategoriesList.append("IP_LOCATION_TYPE_UNSPECIFIED")
+            elif ipCategory == "Anonymous VPN":
+                _ipCategoriesList.append("IP_LOCATION_TYPE_ANONYMOUS_VPN")
+            elif ipCategory == "Hosting Provider":
+                _ipCategoriesList.append("IP_LOCATION_TYPE_HOSTING_PROVIDER")
+            elif ipCategory == "Public Proxy":
+                _ipCategoriesList.append("IP_LOCATION_TYPE_PUBLIC_PROXY")
+            elif ipCategory == "TOR Exit Node":
+                _ipCategoriesList.append("IP_LOCATION_TYPE_TOR_EXIT_NODE")
+            elif ipCategory == "BOT":
+                _ipCategoriesList.append("IP_LOCATION_TYPE_BOT")
+            else:
+                error = f"Unknown ipCategory {ipCategory} specified."
+                demisto.error(error)
```

Comment:
```suggestion
```
It is unnecessary, since in main we already return return_error

Reply:
Done